tslearn/utils.py

```diff
@@ -4,10 +4,47 @@
 
 import numpy
 from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.utils import column_or_1d
+from sklearn.utils.validation import check_is_fitted
+import warnings
 
 __author__ = 'Romain Tavenard romain.tavenard[at]univ-rennes2.fr'
 
 
+def check_dims(X, X_fit=None, extend=True):
+    """Checks whether the dimensions, except the first one which is used
+    for each of the timeseries (which can be variable) of X_fit and X.
+
+    Parameters
+    ----------
+    X : array-like
+        The first array to be compared.
+    X_fit : array-like
+        The second array to be compared, which is created during fit.
+
+    Returns
+    -------
+    nothing
+        Will raise exception if one of both arrays is None or one of the
+        dimensions, except the first, does not match.
+
+    """
+    if X is None:
+        raise ValueError('X is equal to None!')
+
+    if extend and len(X.shape) == 2:
+        warnings.warn('2-Dimensional data passed. Assuming these are '
```

Comment:
This is spammed in the CI :(
Maybe you can add an ignore filterwarning in the [setup.cfg](https://github.com/rtavenar/tslearn/blob/master/setup.cfg) file.

Reply:
Agreed!